app/models/social_messaging/twitter_status.rb

```diff
@@ -11,11 +11,11 @@ def initialize(post)
     def post_to_twitter
       return if post.draft? || post.tweeted
 
-      if ENV['update_twitter_with_post'] == 'true'
-        TwitterClient.update(status)
-        post.tweeted = true
-        post.save
-      end
+      return unless ENV['update_twitter_with_post'] == 'true'
```

Comment:
it's a bit hard to understand easily what's trying to do, maybe:
```ruby
return if ENV.fetch('SKIP_TWITTER_POSTING', false)
```

Reply:
I was just following rubocop on that approach